esx_service/vmci/vmci_client.c

```diff
@@ -349,5 +350,7 @@ Vmci_GetReply(int port, const char* json_request, const char* be_name,
    req.mlen = strnlen(json_request, MAXBUF) + 1;
    req.msg = json_request;
 
+   ans->buf = NULL;
```

Comment:
wait, why here? if you want it to be NULL, the right place to change would be malloc->calloc in esx_vmdkcmd.go.